### PR TITLE
Remove Reference to Chromeless param

### PR DESCRIPTION
**Pull request by apsychogirl**
Fixes - #828 
>Removed the instance of "Chromeless" from [packages/fxa-content-server/app/tests/spec/models/reliers/relier.js] .

### CODE REVIEW

File: packages/fxa-content-server/app/tests/spec/models/reliers/relier.js [hunk 1]
```diff
@@ -110,10 +110,6 @@ describe('models/reliers/relier', function() {
     });
   });
 
-  ['trailhead-1', 'chromeless'].forEach(value => {
```
Comment (vladikoff) [inline packages/fxa-content-server/app/tests/spec/models/reliers/relier.js]: @apsychogirl could you please just remove the `chromeless`, let's keep the trailhead-1 while we still support the `style` query param.

Comment (apsychogirl) [reply packages/fxa-content-server/app/tests/spec/models/reliers/relier.js]: > @apsychogirl could you please just remove the `chromeless`, let's keep the trailhead-1 while we still support the `style` query param.

Should I also remove the `value` query param which was earlier taking the value for `Chromeless` in line 114?

**Comment by vbudhram** [comment]
Hey @apsychogirl, thanks for PR! Before we can merge, can you squash commits into one and update message to follow https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md#git-commit-guidelines.

**Comment by vbudhram** [comment]
@apsychogirl Looks like you will also need to setup GPG signing, ref https://help.github.com/en/github/authenticating-to-github/about-commit-signature-verification#gpg-commit-signature-verification.

**Comment by clouserw** [comment]
@apsychogirl are you able to set up GPG signing for your commits?  If not we can commit this for you.  Thanks for the patch!

**Comment by apsychogirl** [comment]
My laptop just broke... won't be able to sign the commit :(

**Comment by clouserw** [comment]
The source branch for this was removed/unknown so I can't push here.  I opened a new PR with your patch in it @ https://github.com/mozilla/fxa/pull/5000

**Pull request closed without merging**